bit/network/meta.py

```diff
@@ -49,20 +52,26 @@ def __eq__(self, other):
             and self.txid == other.txid
             and self.txindex == other.txindex
             and self.segwit == other.segwit
+            and self.sequence == other.sequence
         )
 
     def __repr__(self):
-        return 'Unspent(amount={}, confirmations={}, script={}, txid={}, txindex={}, segwit={})'.format(
+        return 'Unspent(amount={}, confirmations={}, script={}, txid={}, txindex={}, segwit={}, sequence={})'.format(
             repr(self.amount),
             repr(self.confirmations),
             repr(self.script),
             repr(self.txid),
             repr(self.txindex),
             repr(self.segwit),
+            repr(int.from_bytes(self.sequence, byteorder='little'))
         )
 
     def set_type(self, type, vsize=0):
         self.type = type if type in UNSPENT_TYPES else 'unknown'
         self.vsize = vsize if vsize else UNSPENT_TYPES[self.type]['vsize']
         self.segwit = UNSPENT_TYPES[self.type]['segwit']
         return self
+
+    def opt_in_for_RBF(self):
+        if int.from_bytes(self.sequence, byteorder='little') > 0xFFFFFFFD:
+            self.sequence = 0xFFFFFFFD.to_bytes(4, byteorder='little')
```

Comment:
The `Unspent` object contains string and int values, however sequence is treated like a byte. It is therefore not possible to use this helper function on an `Unspent` object created from its representation (e.g. `Unspent(amount=10000, confirmations=1, script='a9143cf881f889093f87de36d62796e1221832c71d0787', txid='037c95ec975738415f287deca1a4166d4e3b254ae1b7df991cd04ee154401315', txindex=1, segwit=True, sequence=4294967295)`).

However, if this helper function is moved to the `TxIn` object, it should behave correctly as the `TxIn` object contains bytes values.

Reply:
Good suggestion, it's now an integer 7b4d000cb3d318d5d47274f3c6f5f0d054696868